x-pack/plugins/lists/public/b.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+// @ts-ignore
+import * as a from './a';
+
+export const b = 1;
+
+// TEST FILE THAT SHOULD FAIL THE BUILD AND THEN WILL BE REMOVED
```

Comment:
NOTE: This should not pass the CI build system and I will remove them after seeing CI fail for a CI success.